xivo_dao/resources/extension/fixes.py

```diff
@@ -64,28 +55,9 @@ def adjust_for_user(self, extension_id, user_id):
          .update({'type': 'user',
                   'typeval': str(user_id)}))
 
-    def find_incall_id(self, extension_id):
-        return (self.session.query(Incall.id)
-                .join(Extension,
-                      and_(Extension.type == 'incall',
-                           cast(Extension.typeval, Integer) == Incall.id))
-                .filter(Extension.id == extension_id)
-                .scalar())
-
-    def adjust_incall(self, extension_id, incall_id):
-        row = (self.session.query(Extension.exten,
-                                  Extension.context)
-               .filter(Extension.id == extension_id)
-               .first())
-
-        (self.session.query(Incall)
-         .filter(Incall.id == incall_id)
-         .update({'exten': row.exten,
-                  'context': row.context}))
-
     def reset_destination(self, extension_id):
         destination = self.get_destination(extension_id)
-        if destination in ('user', 'incall'):
+        if destination in ('user',):
```

Comment:
how about
```python
if destination == 'user':
```

Reply:
That's what I did, but I revert the change because it was easier to understand the logic by leaving as it. But no objection to use `==` syntax if it annoy you